### PR TITLE
Followable attributes array members can be used for navigation

### LINKED ISSUE #582: [MetadataViewer] Navigation for NeXus attributes
Make the values of any `default`, `signal`, `axes`, etc. attributes clickable in the `MetadataViewer` so users can quickly navigate to the entities.

### PR DESCRIPTION
**Pull request by loichuder**
Fix #582 

Not a huge fan of the resulting code, I may have missed a simpler solution. Perhaps I need my :coffee: :smile: 

**Before:**
image

**After:**
image

### CODE REVIEW

File: src/h5web/metadata-viewer/AttributeValueLink.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <AttributeValueLink onFollowPath={onFollowPath} value={val} />
+            {i < value.length - 1 && <span>,</span>}
+          </Fragment>
+        ))}
+        ]
+      </div>
+    );
+  }
+
+  if (typeof value === 'string' && value !== '.') {
```
Comment (loichuder) [inline src/h5web/metadata-viewer/AttributeValueLink.tsx:29]: `.` is a placeholder and cannot be used for navigation so we might as well put it out of the way.

File: src/h5web/metadata-viewer/AttributesTable.tsx [hunk 2]
```diff
@@ -35,16 +35,8 @@ function AttributesTable(props: Props) {
...
-                  className={styles.attrValueBtn}
-                  type="button"
-                  aria-label={`Inspect ${value}`}
-                  onClick={() => onFollowPath(value)}
-                >
-                  {value}
-                  <FiChevronsRight />
-                </button>
+              {FOLLOWABLE_ATTRS.has(name) ? (
+                <AttributeValueLink onFollowPath={onFollowPath} value={value} />
```
Comment (loichuder) [inline src/h5web/metadata-viewer/AttributesTable.tsx:39]: Open to suggestions on the name

File: src/h5web/metadata-viewer/AttributeValueLink.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        aria-label={`Inspect ${value}`}
+        onClick={() => onFollowPath(value)}
+      >
+        {value}
+        <FiChevronsRight />
+      </button>
+    );
+  }
+
+  return <>{JSON.stringify(value)}</>;
```
Comment (axelboc) [inline src/h5web/metadata-viewer/AttributeValueLink.tsx:43]: No need for the fragment in theory.

**Review by axelboc — APPROVED** [review]
Looks great to me. Can you just add spaces around the characters `[`, `]` and `,`?

File: src/h5web/metadata-viewer/AttributeValueLink.tsx (hunk 3 above)
Comment (loichuder) [reply src/h5web/metadata-viewer/AttributeValueLink.tsx:43]: I cannot remove it as I get `string | Element cannot be used as Element` when using `AttributeValueLink` recursively above.

**Pull request merged**